config/defaults.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     publishMetaData:      'false',
     statsComponent:       undefined,
     awsKeyPrefix:         undefined,
-    iterationInterval:    1000 * 10,
+    iterationInterval:    1000 * 1,
```

Comment:
hmm... Maybe what we need is `minIterationInterval` as a setting for the minimum time between start of iterations.

Hence, if an iteration takes `x` ms, we sleep for `Math.max(minIterationInterval - x, 0)` ms before next iteration. So if iterations takes longer than 75s we don't sleep between them... Just an example...